### ISSUE TITLE
[discovervalues] 'missing' error if a values directory does not contain strings.xml

### ISSUE THREAD

**Issue by liamnichols** [Asana/locheck#16]
Hello, first off all let me say how awesome this tool is! I've run it on our iOS project and it works like a charm and since you have Android support, I also really wanted to try it out on our Android project! 

When doing so, I hit the following issue:

```bash
$ locheck discovervalues ~/Code/GitHub/cookpad/global-android/ui-commons/src/main/res --ignore-missing
Error: Files encountered an error at '/Users/.../global-android/ui-commons/src/main/res/values-en-rGB/strings.xml'.
Reason: missing
```

I'm not too familiar with our Android project, but our base **strings.xml** is in the **values** directory, we then have some regional directories for **values-en-rGB** and so on but we don't localise strings for different regions (at least yet) so the **values-en-rGB/strings.xml** doesn't exist. 

Now I figure this is something relatively easy to patch so I'm happy to try and submit a PR, but first i wanted to check with you on the expected behaviour. Is our project configuration common by not having **strings.xml** in every values directory? I guess I see at least three ways to fix this:

1. Ignore the values directory if it doesn't contain **strings.xml**
2. The above, but only when `--ignore-missing` is set? 
3. A different option for passing locales/directories to ignore? 

Let me know your thoughts! Thanks again 🚀

**Comment by liamnichols** [comment]
After experimenting a bit more, I see that maybe the option here is for me to specify each directory that I do want to include manually? We have >25 languages in the project so it would be a bit cumbersome to do that (but not impossible) so I wonder if it would be beneficial to explore alternative options still?

**Comment by stevelandeyasana** [comment]
I have no idea how common it is, so I'm really glad you reported this problem. :-) It would be trivial to modify `discovervalues` to ignore directories without `strings.xml` files. I think developers are well aware when they don't ship in a given language, so skipping with at most an info log message seems appropriate to me. I'll throw up a PR.

**Comment by stevelandeyasana** [comment]
Will be fixed by #17.

**Comment by liamnichols** [comment]
Great! Thank you so much for this 🚀
